src/pages/index.js

```diff
@@ -82,12 +131,37 @@ function getCardElement(data) {
 
   const cardLikeBtnEl = cardElement.querySelector(".card__like-button");
   cardLikeBtnEl.addEventListener("click", () => {
-    cardLikeBtnEl.classList.toggle("card__like-button_active");
+    const isLiked = cardLikeBtnEl.classList.contains(
+      "card__like-button_active"
+    );
```

Comment:
* When I reload the page, all **red** hearts become **transparent** again.


Please, check `  if (data.isLiked) `


```js

  if (data.isLiked) {
    cardLikeBtnEl.classList.add("card__like-button_active");
  }
```